### DISCUSSION TITLE
Use code writen by python from handle in component

### DISCUSSION THREAD

**Discussion by zyrfyxx** [nasa/fprime#1880] [Q&A]
I want to use a function weiten by python in a cmd handle in component,so I use the lib python3.7m. So in order to link lib python3.7m, I add this to the CmakeLists.txt in component:
```
include_directories(/usr/include/python3.7m)
find_library(PYTHON3.7M python3.7m "/usr/lib/x86_64-linux-gnu/")
add_library(python3.7m SHARED IMPORTED GLOBAL)
set_property(TARGET python3.7m PROPERTY IMPORTED_LOCATION ${PYTHON3.7M})
```
After this, I run fprime-build and there is no error.
In the handle, I use the code:
`Py_Initialize();`
But when I use the cmd in fprime-gds, the fprime-gds doesn't work and the green point on the right top of the gds interface become a red 'X'.

**Comment by LeStarch** [comment]
It is likely crashing, look for crash output in the terminal.  Have you looked at `pybind11`?  It makes working with an embedded interpreter much easier.
